src/download.jl

```diff
@@ -0,0 +1,51 @@
+"""
+dataurl(dataset_code)
+This function takes in a dataset code of format "GADM/<country_code>"
+Returns the URL of the gpkg dataset of the country
+"""
+function dataurl(dataset_code)
+
+    # Splits GADM/IND to GADM, IND
+    dataset_provider, country_code = split(dataset_code, "/")
+    
+    # Only accepts GADM Dataset Provider
+    if dataset_provider != "GADM"
+        error("❌ Dataset Provider $dataset_provider not supported. 💡 Please try \"GADM/<Code>\".")
+    end
+    
+    @info "✅ fetching $country_code's data from GADM\n"
+
+    dataset_url = "https://biogeo.ucdavis.edu/data/gadm3.6/gpkg/gadm36_$(country_code)_gpkg.zip"
+
+    return dataset_url 
```

Comment:
The last statement is returned by default. Not many packages adopted the convention to use the `return` keyword at the end. It is only used to return prematurely before the end of the function.

These 3 lines could be replaced by the string:

```julia
"https://biogeo.ucdavis.edu/data/gadm3.6/gpkg/gadm36_$(country_code)_gpkg.zip"
```